app/helpers/users_circuitverse_helper.rb

```diff
@@ -4,7 +4,9 @@ module UsersCircuitverseHelper
   def get_priority_countries
     priority_countries = ["IN"]
     geo_contry = Geocoder.search(request.remote_ip).first&.country
-    priority_countries.prepend(geo_contry) unless geo_contry.nil?
+    if geo_contry != "IN"
```

Comment:
```suggestion
    if priority_countries.include?(geo_country)
```
make it more generic